src/store.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useReducer, useEffect, useMemo, useState } from 'react';
+import { Todo } from './types/todo';
+
+export const LOCAL_STOR_KEY = 'todos';
+export const storedTodos = localStorage.getItem(LOCAL_STOR_KEY);
```

Comment:
```suggestion
export const storedTodos = localStorage.getItem(LOCAL_STOR_KEY);

if (!storedTodos) {
  localStorage.setItem(LOCAL_STOR_KEY, JSON.stringify([]));
}
```
Consider setting an empty array on the first load of the app